ServiceWorkerGlobalScope/ServiceWorkerGlobalScope_contentdelete_event.html

```diff
@@ -0,0 +1,82 @@
+<div>{{draft}}{{DefaultAPISidebar("Service Worker API")}}</div>
+
+<p class="summary">The <strong><code>contentdelete</code></strong> event of the {{domxref("ServiceWorkerGlobalScope")}} interface is fired when an item is removed from the indexed content via the user agent. </p>
+
+<table class="properties">
+  <tbody>
+   <tr>
+    <th scope="row">Bubbles</th>
+    <td>No</td>
+   </tr>
+   <tr>
+    <th scope="row">Cancelable</th>
+    <td>No</td>
+   </tr>
+   <tr>
+    <th scope="row">Interface</th>
+    <td>{{domxref("ContentIndexEvent")}}</td>
+   </tr>
+   <tr>
+    <th scope="row">Event handler property</th>
+    <td><code><a href="/en-US/docs/Web/API/ServiceWorkerGlobalScope/oncontentdelete">oncontentdelete</a></code></td>
```

Comment:
```suggestion
    <td><code>{{domxref('ServiceWorkerGlobalScope.oncontentdelete')}}</code></td>
```

Reply:
I'll add this to BCD.

Reply:
Vinyl's email on the 17th said she wasn't adding Content Index yet because of support. From what I can see it's not in the bcd repo. My own code still only works in Canary, but status and the fugu spreadsheet seem to point at it being released into stable already - need to confirm and am happy to do so under your direction :)